apps/core/src/interfaces/transactions.interfaces.ts

```diff
@@ -16,6 +16,8 @@ export enum TransactionAction {
     Transaction = 'Transaction',
     Staked = 'Staked',
     Unstaked = 'Unstaked',
+    TimelockedStaked = 'TimelockedStaked',
+    TimelockedUnstaked = 'TimelockedUnstaked',
```

Comment:
we should add a space here so the labels are not displayed like this
![image](https://github.com/user-attachments/assets/72d8c6eb-e052-4fa7-8a3c-05633c662de7)